deps/3rdparty/libdispatch/build/libdispatch.vcxproj

```diff
@@ -28,36 +28,36 @@
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
     <ConvergedProjectType>CodeSharingDll</ConvergedProjectType>
     <OutputName>libdispatch</OutputName>
-    <WindowsTargetPlatformVersion>10.0.14393.0</WindowsTargetPlatformVersion>
-    <WindowsTargetPlatformMinVersion>10.0.10240.0</WindowsTargetPlatformMinVersion>
+    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformMinVersion>10.0.15063.0</WindowsTargetPlatformMinVersion>
```

Comment:
>n>10.0.15063.0</WindowsTargetP [](start = 35, length = 30)

the minimum version should not be 15063.

Reply:
(the minimum version should be whatever minimum version the rest of the SDK uses; perhaps 14393?)

---
In reply to: [141440251](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141440251) [](ancestors = 141440251)